src/sst/elements/vanadis/decoder/vriscv64decoder.h

```diff
@@ -712,6 +730,16 @@ class VanadisRISCV64Decoder : public VanadisDecoder
                         bundle->addInstruction(new VanadisOrInstruction(ins_address, hw_thr, options, rd, rs1, rs2));
                         decode_fault = false;
                     } break;
+                    case 0x1:
+                    {
+                        // REM
+                        output->verbose(
+                            CALL_INFO, 16, 0, "-----> REM %" PRIu16 " <- %" PRIu16 " %% %" PRIu16 "\n", rd, rs1, rs2);
+                        bundle->addInstruction(
+                            new VanadisModuloInstruction<int64_t>(
+                                ins_address, hw_thr, options, rd, rs1, rs2));
```

Comment:
`REM` and `REMU` are 32bit instructions.

Reply:
Are you sure? XLEN should be 64. I think REM[U]W are the 32b variations for RV64.
Format
    rem rd,rs1,rs2
Description
    perform an XLEN bits by XLEN bits signed integer reminder of rs1 by rs2.
Implementation
    x[rd] = x[rs1] %s x[rs2]